packages/zowe-explorer/src/dataset/DatasetTree.ts

```diff
@@ -683,25 +681,24 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
      */
     public async removeFavProfile(profileName: string, userSelected: boolean) {
         // If user selected the "Remove profile from Favorites option", confirm they are okay with deleting all favorited items for that profile.
+        let cancelled = false;
         if (userSelected) {
             const checkConfirmation = localize(
                 "removeFavProfile.confirm",
                 "This will remove all favorited Data Sets items for profile {0}. Continue?",
                 profileName
             );
             const continueRemove = localize("removeFavProfile.continue", "Continue");
-            const cancelRemove = localize("removeFavProfile.cancel", "Cancel");
-            const quickPickOptions: vscode.QuickPickOptions = {
-                placeHolder: checkConfirmation,
-                ignoreFocusOut: true,
-                canPickMany: false,
-            };
-            // If user did not select "Continue", do nothing.
-            if (
-                (await vscode.window.showQuickPick([continueRemove, cancelRemove], quickPickOptions)) !== continueRemove
-            ) {
-                return;
-            }
+            await vscode.window
+                .showWarningMessage(checkConfirmation, { modal: true }, ...[continueRemove])
+                .then((selection) => {
+                    if (!selection || selection === "Cancel") {
```

Comment:
At first I was worried about removing the localization from the selection, but it looks like this condition handles other languages. I tried it with spanish language and when I clicked "Cancelar" it returned undefined so it still counted as cancelled.

Reply:
@crawr I am wondering if the `Cancel` button will always return undefined since it's not a passed button into the confirmation, the modal:true confirmation boxes have a built in Cancel button.